README.md

```diff
@@ -113,7 +113,15 @@ See the [developer guide](https://stfc.github.io/aiida-mlip/developer_guide/inde
 
 ## License
 
-BSD 3-Clause License
+[BSD 3-Clause License](LICENSE)
+
+## Funding
+
+Contributors to this project were funded by
+
+- [![PSDI](docs/source/images/psdi-100.webp)](https://www.psdi.ac.uk/)
```

Comment:
Two minor points: 

I'd consider getting rid of the bullet points, as it looks a little odd to me with the images.

Any chance we have higher resolution images immediately to hand? I think it's fine if not.

Reply:
bulleting removed... ideally we will get svg for all of them. but high resolution baloons the repo for no benefit